docs/index.html

```diff
@@ -489,15 +501,18 @@ <h5>SSH Agent integration</h5>
 
         <dt id="faq-ssh-agent-pageant"><a href="#faq-ssh-agent-pageant">Why is Pageant refusing my keys?</a></dt>
```

Comment:
This should probably be expanded to Pageant and OpenSSH (both versions?) if what I noted below is still true.

Reply:
Will check – I didn't put anything in here because I simply didn't verify that behaviour at the time and the consensus was not to expand the FAQs too much.

Reply:
I added a sentence explaining that Windows OpenSSH is not compatible with confirm-on-use and timeouts after verifying the behaviour locally.